cico_build_deploy.sh

```diff
@@ -2,8 +2,12 @@
 
 . cico_setup.sh
 
-cico_setup;
-
-run_tests_without_coverage;
+load_jenkins_vars
+ run_tests_without_coverage;	if [ ! -f .cico-prepare ]; then
```

Comment:
`run_tests_without_coverage` called twice here; I guess you forgot to delete it from here ??  It only should be inside `if`

Reply:
ah yes, that's the result of a too quick copy/paste/commit/push :/